submissions/semenjob/style.css

```diff
@@ -0,0 +1,112 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+img {
```

Comment:
it is a bad practice to style by HTML tag
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css